### ISSUE TITLE
Accessibility in Dark Mode

### ISSUE THREAD

**Issue by mweidling** [OCR-D/quiver-frontend#26]
The labels are hardly readable in dark mode:

![Screenshot from 2022-12-14 13-44-49](https://user-images.githubusercontent.com/13831557/207601620-20d37663-d85a-4b18-98a8-6f81dd113bc1.png)

Maybe dark mode should be investigated overall regarding accessibility.

**Comment by kba** [comment]
Yes, please, I have to switch browsers to use quiver currently. Help us dark mode afficionados :-)

**Comment by paulpestov** [comment]
Resolved in 9c4a9f1

**Comment by kba** [comment]
![image](https://user-images.githubusercontent.com/273367/213731660-1f10372f-75f5-484b-b320-2e36155dae93.png)

Nice! Can be closed now IMHO.
